Configuration key api_mastodon_banner added

Quix0r opened the pull request:
Added:
- config key 'config', 'api_mastodon_banner', see discussion at https://b65.me.in/display/67fef576-1564-a27a-405e-c17705255720
- the administrator can now customize it for Mastodon contacts
- credits goes to `@betamax65@b65.me.in`

Quix0r committed:
Added:
- config key 'config', 'api_mastodon_banner', see discussion at https://b65.me.in/display/67fef576-1564-a27a-405e-c17705255720
- the administrator can now customize it for Mastodon contacts
- credits goes to @betamax65@b65.me.in

MrPetovan commented:
We haven't had a discussion about configuration key grouping, but in my opinion the `config` group should be deprecated and eventually phased out, because it doesn't provide any information. `system` is also a poor catch-all, and I would like to see meaningful groups created and used by new configuration keys.

Quix0r commented:
Yes, I took this group over from the original author and I told him already that he might need to change his configuration file. I think the "group" should be here e.g. `api` or and then not `api_mastodon_banner` but `mastodon_banner` instead?

MrPetovan commented:
`api.mastodon_banner` sounds wonderful.

Quix0r committed:
Changes:
- renamed 'api_mastodon_banner' to 'mastodon_banner' and moved it to 'api'
  configuration group

tobiasd commented:
This is per instance right? So no user setting so far.

Quix0r commented:
> This is per instance right? So no user setting so far.

Yes, it looks like. And old instance entries are not affected by it.
